ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Config.scala

```diff
@@ -40,5 +42,7 @@ object Config {
     included = Set.empty,
     listTests = false,
     allTests = false,
+    waitForParties = true,
```

Comment:
Perhaps we could make this `false` and just enable it for the Canton tests.

Reply:
I would prefer to preserve the current semantics until we have a shared understanding of what is the behavior that we want.

Reply:
Well, we only added the wait for Canton in the first place. 🙂

Reply:
Because it makes sense for distributed participants that have to be aware of each other's party.